api-extractor/gulpfile.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const build = require('@microsoft/node-library-build');
+
+const tslintCommon = require('@microsoft/sp-tslint-rules');
```

Comment:
Looking at tslint.json:

```json
  "rules": {
    "prefer-const": true,
    "react-a11y-aria-unsupported-elements": true,
    "react-a11y-event-has-role": true,
    "react-a11y-image-button-has-alt": true,
    "react-a11y-img-has-alt": true,
    "react-a11y-lang": true,
    "react-a11y-anchors": false,
    "react-a11y-meta": true,
    "react-a11y-props": true,
    "react-a11y-proptypes": true,
    "react-a11y-role-has-required-aria-props": true,
    "react-a11y-role-supports-aria-props": true,
    "react-a11y-role": true,
    "react-a11y-tabindex-no-positive": true,
    "react-a11y-titles": true
  }
```

Is prefer-const not enabled in GCB?  That is a pretty fundamental rule -- can we enable it somehow?